config.json

```diff
@@ -1764,6 +1764,15 @@
         ],
         "difficulty": 5,
         "topics": ["algorithms", "logic"]
+      },
+      {
+        "slug": "satellite",
+        "name": "Satellite",
+        "uuid": "bfc08dea-2a6b-4696-be20-9b55e282388d",
+        "practices": [],
+        "prerequisites": ["arrays", "sets", "objects", "conditionals"],
```

Comment:
* `sets` and nice here but not needed as prerequisite I think. A student could also use some other way to check for duplicates.
* I would add `array-transformations` to the list as I think those are needed to a reasonable solution (e.g. like yours).
* What do you think about adding `recursion`? Seems like a good fit to me.